Meadowlark-js/packages/meadowlark-core/src/Environment.ts

```diff
@@ -14,3 +20,35 @@ export function getValueFromEnvironment(key: EnvironmentVariable): string {
 
   throw new Error(`Environment variable '${key}' has not been setup properly.`);
 }
+
+/*
```

Comment:
Because I'm parsing the school years out from a single string, I wanted the ability to cache them for quick lookup next time they're needed.